nativeunwind/elfunwindinfo/stackdeltaextraction_test.go

```diff
@@ -157,3 +160,63 @@ func TestExtractStackDeltasFromFilename(t *testing.T) {
 	}
 	require.Equal(t, data.Deltas[:len(firstDeltas)], firstDeltas)
 }
+
+func TestLookupFDE(t *testing.T) {
```

Comment:
last minute nit: perhaps worth a a separate `elfehframetable_test.go`? Or just remove the whole file to `ehframe_test.go` since now its testing more than stack deltas.